crm_phonecall/security/crm_security.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="group_scheduled_calls" model="res.groups">
```

Comment:
This security group is not used in this module, so we could delete it if not required.

Reply:
it's is used in please check views/crm_phonecall_view.xml